prometheus_juju_exporter/collector.py

```diff
@@ -140,6 +140,33 @@ def _get_machine_type(self, machine: Dict) -> MachineType:
 
         return MachineType.METAL
 
+    def _get_machine_sensible_hostname(self, machine: Dict) -> str:
```

Comment:
non-blocker: Perhaps a `_get_machine_identifier()` would be a better name? This is very much a nit pick based on my opinion but the word "sensible" is bothering me in the function name.

Reply:
I like your idea, it's less snarky and more professional :D I will push an update tomorrow thanks!